devlab/tests/config.py

```diff
@@ -169,3 +153,55 @@
 GLANCE_CLIENT_VERSION = '1'
 NEUTRON_CLIENT_VERSION = '2.0'
 CINDER_CLIENT_VERSION = '1'
+
+
+# Keypairs to create/delete
+# Connected to user's list
+keypairs = [
```

Comment:
Why do we need to hardcode things like this? As I understand we can generate keys when we need. Or do you need this keys as a pattern for assertion in test?